src/components/klage-eller-anke/klage-eller-anke-innsending.tsx

```diff
@@ -100,7 +105,7 @@ const DigitalContent = (props: DigitalContentProps) => {
     }
     return (
         <MarginContainer>
-            <KlageLinkPanel href={'/klage?tema=' + props.tema} border>
+            <KlageLinkPanel href={'/klage?tema=' + props.tema + '&ytelse=' + props.ytelse} border>
```

Comment:
Denne sender path-versjonen av ytelsen til skjema?

Reply:
Ja, tenkte det var det beste, men da må vi evt parse den i skjemakomponenten, ellers får vi titler som "Klage på vedtak for foreldrepenger-engangsstønad-svangerskapspenger".

Tenker du det kan være bedre å sende selve tittelen? Hvordan blir det med spaces osv?

Reply:
Tanken var å behandle dette som fritekst. Da må vi (og andre løsninger) URL-enkode teksten, ja.
Dette er bare for å sende tittelen til skjema, ikke for routing i skjemavelgeren.

Reply:
Ok :) Fixed. Sender tittelen nå, ikke path.

![image](https://user-images.githubusercontent.com/19277989/96267305-f055de80-0fc7-11eb-86bc-65cab3cad23b.png)

Reply:
Ser bra ut :)